components/ord-service/src/main/java/com/sap/cloud/cmp/ord/service/storage/model/APISpecificationEntity.java

```diff
@@ -25,6 +30,14 @@
     @Column(name = "spec_data", length = Integer.MAX_VALUE)
     private String specData;
 
+    @EdmProtectedBy(name = "tenant_id")
+    @EdmIgnore
+    @Column(name = "tenant_id", length = 256)
+    @Convert("uuidConverter")
+    @TypeConverter(name = "uuidConverter", dataType = Object.class, objectType = UUID.class)
```

Comment:
Is those needed since apiSpecifications are not part of the OData API and the tenancy is based on the repository method?

Reply:
If you mean the whole `UUID tenant` field, it is needed so that the JPA repository can work properly.

If you mean the two `@Edm...` annotations, we added them to be on the safe side. Without them the flow works as expected.

Reply:
Yes, I meant the 2 Еdm annotations. They are unneeded since the whole class has `@EdmIgnore`, but it is fine to leave them also